server/start.go

```diff
@@ -244,6 +247,53 @@ func startStandAlone(ctx *Context, appCreator types.AppCreator) error {
 	return WaitForQuitSignals()
 }
 
+func restoreLocalSnapshot(ctx *Context, app types.Application, ssRestoreHeight uint64) error {
+	ctx.Logger.Info(fmt.Sprintf("Local Statesync Restore Snapshot With Height: %d", ssRestoreHeight))
+	ctx.Config.StateSync.RestoreHeight = ssRestoreHeight // Should be a one-shot command
```

Comment:
Why do we need to update the config?

Reply:
Will remove.

Reply:
FYI: Used here in the tendermint layer to detect a statesync, but there might be a way to remove it.

https://github.com/tendermint/tendermint/pull/9541/files#diff-4eaef000463512634f080db6f15d6aeb1b856e995a148bbeea5ee94758210fe1R281

Reply:
Discussing with the tendermint maintainers if there's a more preferred way to send a signal that cosmos-sdk performed a "Lazy State Sync".  

They are asking to consider a slightly different code path which might require a variable in a different place (or no variable at all).